helper/schema/provider_test.go

```diff
@@ -180,7 +181,7 @@ func TestProviderConfigure(t *testing.T) {
 		c := terraform.NewResourceConfigRaw(tc.Config)
 		diags := tc.P.Configure(context.Background(), c)
 		if diags.HasError() != tc.Err {
-			t.Fatalf("%d: %s", i, errorDiags(diags))
+			t.Fatalf("%d: %s", i, diagutils.ErrorDiags(diags))
```

Comment:
Please delete `diagnostic_test.go` I think you will need to port a few lines to `diagutils.Warnings`

Reply:
Oh you did move the file sorry.... weird is `warnDiags` not used at all?